pkgs/by-name/el/elia/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "elia";
+  version = "1.7.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "darrenburns";
+    repo = "elia";
+    rev = version;
```

Helps to handle branch/tag confusion.

```suggestion
    rev = "refs/tags/${version}";
```

---

Thanks, I've made all of the relevant changes but it seems the package will not be able to build until this is merged [darrenburns/elia#47](https://github.com/darrenburns/elia/pull/47), due to the version of the textual dependency